examples/gallery/index.html

```diff
@@ -36,11 +43,38 @@ <h1>Slint Gallery</h1>
     </a>
   </p>
   <script type="module">
-    import init from './pkg/gallery.js';
-    init().finally(() => {
-      document.getElementById("spinner").remove();
-    });
+    var fluent = `./pkg/fluent/gallery.js`;
+    var material = "./pkg/material/gallery.js";
+
+    function initGallery(gallery) {
+      document.getElementById("spinner").hidden = false;
+      document.getElementById("canvas").hidden = true;
+
+      import(gallery).then(module => {
+        module.default().finally(() => {
+          document.getElementById("canvas").hidden = false;
+          document.getElementById("spinner").hidden = true;
+        });
+      })
+    }
+
+    initGallery(fluent);
+
+    var styleSelection = document.getElementById("style-selection");
+    styleSelection.onchange = function (ev) {
+      var selectedStyle = styleSelection[styleSelection.selectedIndex].innerHTML;
```

Comment:
Minor suggestion. Doubt we'll put markup in there, but just to be on the safe side :)

```suggestion
      var selectedStyle = styleSelection[styleSelection.selectedIndex].innerText;
```